sql-cli/pyproject.toml

```diff
@@ -1,30 +1,16 @@
-[build-system]
-requires = ["flit_core ~=3.2"]
-build-backend = "flit_core.buildapi"
-
 [project]
 name = "astro-sql-cli"
-dynamic = ["version"]
-description = """
-Astro SDK allows rapid and clean development of {Extract, Load, Transform} workflows using Python and SQL, powered by Apache Airflow.
-"""
-
+version = "0.0.0"
+description = "Empower analysts to build workflows to transform data using SQL"
 authors = [
     {name = "Astronomer", email = "humans@astronomer.io"},
 ]
 readme = "README.md"
 license = {file = "LICENSE"}
-
 requires-python = ">=3.7"
-dependencies = [
-    "astro-sdk-python[all]>=1.0.0",
```

Comment:
We'll need this to validate the DAGs we've generated are valid. But this is out of the scope of the current ticket.
@feluelle, can we add a ticket for us to test the generated DAGs using a similar strategy to the Python SDK?

Reply:
This probably makes sense to be a test dependency